src/scry/protocol/hover.cr

```diff
@@ -0,0 +1,13 @@
+require "json"
+
+module Scry
+  struct Hover
+    JSON.mapping({
+      contents: MarkupContent,
+      range:    Range?,
+    }, true)
```

Comment:
Use named arg `static: true` here, 'cause by just reading `true` I have no idea what it refers to for a json mapping, I think it's better with the named arg

Reply:
Also, why only this mapping have this static check?